src/api/controllers/moderation.controller.js

```diff
@@ -66,6 +67,22 @@ exports.moderatePost = async (req, res, next) => {
       });
     }
 
+    // Generate metadata of the notification
+    const metadata = {
+      permlink: post.permlink,
+      category: post.category,
+      author: post.author,
+    };
+
+    // Retrieve back socketio instanace
+    const io = req.app.get('socketio');
+
+    // Decide the type of the notification
+    const notificationType = approved === true || approved === 'true' ? 'POST_APPROVED' : 'POST_NOT_APPROVED';
```

Comment:
`approved? 'POST_APPROVED': 'POST_NOT_APPROVED'`

will it not work

Reply:
explain??

Reply:
what is the boolean filter is bypased and any value other than true is passed?